src/fs/directory.rs

```diff
@@ -1,5 +1,4 @@
-use crate::driver::Op;
-
+use crate::runtime::driver::op::Op;
```

Comment:
Again, JTOL. I've never noticed this remove_dir function before. Odd we don't have other directory functions yet and odd to me the argument to this isn't like the arg to open in file.rs.